webapp/portlet/src/main/webapp/idm-users-management/components/UsersManagementToolbar.vue

```diff
@@ -114,9 +115,10 @@ export default {
     multiSelectAction(action) {
       this.$root.$emit('multiSelectAction', action);
     },
-    applyAdvancedFilter(userType) {
+    applyAdvancedFilter(isConnected, userType) {
+      this.isConnected = isConnected;
       this.userType = userType;
-      this.numberOfFilters = userType != null ? 1 : 0;
+      this.numberOfFilters = (userType != null) + (isConnected != null);
```

Comment:
I prefer here that you increment the numberOfFilters variable when the condition is true instead of boolean addition

Reply:
@rdenarie to be confirmed for @AzmiTouil

Reply:
perhaps we should create a countFiltersMethods, and yes, I prefer a real count instead of a boolean addition